models/common/job_component_description.go

```diff
@@ -1,17 +1,15 @@
 package common
 
-import v1 "github.com/equinor/radix-operator/pkg/apis/radix/v1"
-
 // RadixJobComponentConfig holds description of RadixJobComponent
 type RadixJobComponentConfig struct {
 	// Resource describes the compute resource requirements.
 	//
 	// required: false
-	Resources *v1.ResourceRequirements `json:"resources,omitempty"`
+	Resources *Resources `json:"resources,omitempty"`
 	// Node defines node attributes, where container should be scheduled
 	//
 	// required: false
-	Node *v1.RadixNode `json:"node,omitempty"`
+	Node *Node `json:"node,omitempty"`
```

Comment:
Maybe this need to consistent with radix-operator type https://github.com/equinor/radix-operator/blob/6c95cb419117d8ffa948fb9802230805958d86d3/pkg/apis/radix/v1/radixapptypes.go#L434

Reply:
What is inconsistent?